### ISSUE TITLE
123series.ru: breakage

### ISSUE THREAD

**Issue by speanc** [uBlockOrigin/uAssets#14256]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://123series.ru/`

### Category

breakage

### Describe the issue

When uBlock is disabled, I am allowed to watch content as normal. With uBlock enabled, while the page still loads, it seems to also block the content that I am trying to watch. I've dealt with this by subjecting myself to ads for about a year or so in hopes it was a temporary issue, but I am nearing the end of my rope. I've been trying to figure out what exactly it's blocking for a while, but I haven't been able to get anywhere. Selecting different servers to stream from gives me the same issue as well, and from what I can tell it stems from it not being able to grab the video itself.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

https://i.imgur.com/A8YN2zM.png
</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.43.0
Chromium: 102
filterset (summary): 
  network: 98427
  cosmetic: 67231
  scriptlet: 17566
  html: 0
listset (total-discarded, last updated): 
  added: 
    RUS-0: 40333-9, 18m
  default: 
    user-filters: 87-10, never
    easylist: 69167-66, 18m
    easyprivacy: 27481-223, 18m
    plowe-0: 3640-782, 18m
    ublock-abuse: 76-1, 18m
    ublock-badware: 4171-92, 18m
    ublock-filters: 32383-147, 18m
    ublock-privacy: 230-3, 18m
    ublock-quick-fixes: 347-4, 18m
    ublock-unbreak: 1851-42, 18m
    urlhaus-1: 5003-0, 18m
filterset (user): [array of 85 redacted]
trustedset: 
  added: [array of 85 redacted]
  removed: 
    about-scheme
    chrome-extension-scheme
    chrome-scheme
switchRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 318 ms (selfie)
  maxAssetCacheWait: 333 ms
popupPanel: 
  blocked: 4
  blockedDetails: 
    addthis.com: 1
    amung.us: 1
    likedstring.com: 2
```
</details>

**Comment by gorhill** [comment]
Works fine here -- using uBO with default settings/lists, and no other content blocker. You have a lot of custom filters, did you really try as demanded?

> - [x] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules

**Comment by speanc** [comment]
That's strange! I first recognized it as an issue when I installed uBlock for a family member, and she had the same issue with a fresh version on that site. However, what she didn't tell me, was that eventually it started working. The custom filters/rules I used were all specific to unrelated sites, so had I assumed that they wouldn't contribute to any issues. Like a fool, I neglected to give a full clean install a shot. Gonna keep an eye on what I do for filters/rules coming up, and keep my fingers out of the deeper workings of the extension.

I will forever live with the shadow of this error that I made, that wasted your time. I appreciate the assistance, all is working now!
